views/partials/posts/post.hbs

```diff
@@ -18,30 +18,36 @@
       <img src="{{post.user.image}}" class="avatar">
     {{/if}}
 
-    {{#if post.currentUser}}
-      <button class="edit-button" id="editBtn-{{post.id}}">Edit</button>
-      <form action="/posts/{{post.id}}/delete" method="POST">
-        <input type="hidden" name="_method" value="DELETE">
-        <input type="submit" value="Delete" onclick="return confirm('Are you sure you want to delete this post?')">
-      </form>
-    {{/if}}
 
     <ul class="post-comment" id="comments-{{post.id}}">
-      {{> posts/new_comment post}}
-      {{> posts/likes post}}
       {{#each post.comments}}
         {{> posts/comment this}}
       {{/each}}
+      {{> posts/new_comment post}}
     </ul>
   </div>
-</li>
+</div>
+
+{{> posts/likes post}}
+
+<div class="edit-delete">
+    {{#if post.currentUser}}
+      <button class="edit-button btn btn-primary" id="editBtn-{{post.id}}">Edit</button>
+      <form action="/posts/{{post.id}}/delete" method="POST">
+        <input type="hidden" name="_method" value="DELETE">
+        <input type="submit" id="submit" class="btn btn-primary" value="Delete" onclick="return confirm('Are you sure you want to delete this post?')">
+      </form>
+    {{/if}}
+</div>
+
+<div class="post-container-gap"></div>
 
 <div id="editModal-{{post._id}}" class="modal">
   <div class="modal-content">
     <form class="updateForm" action="/posts/{{post._id}}/update" method="post" enctype="application/x-www-form-urlencoded">
-      <label for="message">Message:</label>
-      <textarea id="message" name="message">{{post.message}}</textarea><br/>
-      <input type="submit" value="Update Post"/>
+      <label><h2 class="titles">Edit message:</h2></label><br>
+      <textarea class="modal-message" name="message" rows="4">{{post.message}}</textarea><br/><br>
```

Comment:
might need _id="message"_ here after all for modal to work as updateHandler needs _.message_ (could change to _.editMessage_ everywhere to make it unique – the problem with is it's the same id as in the actual new post textarea, so cypress tests are getting confused about which one to pick)